x-team/xp-ui
# Update JobsGrid component

**Release Type:** *Non-breaking change*

Fixes https://x-team-internal.atlassian.net/browse/XP-3365

## Description

Update JobsGrid component

## Checklist


## Steps to Test or Reproduce

- Run `npm start`
- Go to http://localhost:9001/?path=/story/ui-components-xp-registration-jobsgrid--default
- Also check http://localhost:9001/?path=/story/ui-components-xp-registration-jobspagelayout--default-usage-full-composition

## Impacted Areas in Application

`JobsGrid`

## Screenshots

<img width="1680" alt="Screen Shot 2019-10-08 at 14 50 29" src="https://user-images.githubusercontent.com/131859/66421384-25efe100-e9de-11e9-945e-455bc8448cd0.png">

<img width="1679" alt="Screen Shot 2019-10-08 at 14 54 12" src="https://user-images.githubusercontent.com/131859/66421391-27b9a480-e9de-11e9-8de9-a38951a1f498.png">
